Week3/mysql.sql

```diff
@@ -0,0 +1,70 @@
+CREATE database bag;
+use bag;
+
+
+CREATE TABLE `product` ( 
+    `id` int(10) NOT NULL AUTO_INCREMENT, 
+    `name` varchar(45) NOT NULL , 
+    `price` double ,
```

Comment:
I think we talked about this in class - but if we didn't, you should read up on why 'double' types, or any floating-point type, is a really bad choice for storing fixed-point numbers like prices.